pages/treeherder.py

```diff
@@ -22,12 +22,23 @@ class TreeherderPage(Base):
     _result_sets_locator = (By.CSS_SELECTOR, '.result-set:not(.row)')
     _unchecked_repos_links_locator = (By.CSS_SELECTOR, '#repoLabel + .dropdown-menu .dropdown-checkbox:not([checked]) + .dropdown-link')
     _unclassified_failure_count_locator = (By.ID, 'unclassified-failure-count')
+    _action_menu_dropdown_locator = (By.CSS_SELECTOR, ".th-view-content > ng-view:nth-child(1) > div:nth-child(2) > div:nth-child(1) > span:nth-child(3) > th-action-button:nth-child(4) > span:nth-child(1) > button:nth-child(1)")
```

Comment:
This locator is really fragile. I added this dropdown locator in another test, I would recommend that you look at this for ways to shorten your locators:
https://github.com/rbillings/treeherder-tests/blob/add_new_jobs/pages/treeherder.py#L95

Also, the locators for lines 25,26, and 27 will need to be moved to the ResultSet class.